ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt

```diff
@@ -21,25 +35,29 @@ import com.example.android.wearable.composeadvanced.R
  * Displays a value by using a Stepper or a Slider
  */
 @Composable
-fun ValueDisplayScreen(
+fun UserInputComponentsScreen(
     value: Int,
     onClickStepper: () -> Unit,
     onClickSlider: () -> Unit
 ) {
     Column(
         modifier = Modifier
             .fillMaxSize()
-            .padding(top = 10.dp)
+            .padding(vertical = 10.dp)
             .padding(horizontal = 10.dp),
-        verticalArrangement = Arrangement.Center
+        verticalArrangement = Arrangement.spacedBy(4.dp, Alignment.CenterVertically)
```

Comment:
Is this over a 100 chars? Wasn't sure.

Reply:
It looks long, but i made sure each line doesn't exceed the line length marker, which i set to 100 in intellij

Reply:
Tip: use Cmd + Opt + L in file to auto-format (and also set max line length to 100 chars in AS settings: Preferences -> Code Style => Hard wrap at ).